spec/whitehall/unpublish_document_spec.rb

```diff
@@ -0,0 +1,35 @@
+feature "Unpublishing a document on Whitehall", new: true, whitehall: true, government_frontend: true do
```

Comment:
Should we be naming the type of unpublishing in the test name and title (since we have a few flavours of unpublishing). 

Also don't think this is a government_frontend test

Reply:
Yeah, I was erring on the side of caution with ` government_frontend `.   The test makes a request which will route through to ` government_frontend ` so a 500 there could break this test.  On the flipside, there is no specific assertion against the rendered content, so it isn't a strong coupling.